docs/src/main/sphinx/connector/hive.rst

```diff
@@ -521,6 +529,39 @@ Property Name                                                   Description
 
 ``hive.metastore.thrift.client.ssl.trust-certificate-password`` Password for the trust store
 
+``hive.metastore.thrift.delete-files-on-drop``                  When a managed table is dropped, ensure Trino deletes
+                                                                all data for the table. The Hive metastore is responsible
```

Comment:
Is that true .. does the HMS actually run the delete of the actual data? I thought it just manages the metadata...

Reply:
In case of dealing with tables of type `MANAGED_TABLE` , when dropping the table, HMS is responsible of dropping the content of the table as well. Do note that AWS Glue metastore does not provide the same functionality when dropping tables.